package.json

```diff
@@ -22,25 +22,25 @@
   },
   "homepage": "https://github.com/bendrucker/proxyquire-universal",
   "devDependencies": {
-    "array-includes": "~2.0.0",
+    "array-includes": "~3.0.3",
     "browserify": "latest",
-    "detective": "~4.1.0",
+    "detective": "~5.1.0",
     "escape-string-regexp": "~1.0.3",
     "pff": "~1.0.0",
-    "proxyquire": "^1.6.0",
+    "proxyquire": "^2.1.0",
     "proxyquireify": "^3.0.0",
-    "standard": "^4.0.0",
-    "stream-to-promise": "1",
-    "tape": "~4.0.0",
+    "standard": "^12.0.1",
+    "stream-to-promise": "^2.2.0",
+    "tape": "~4.9.1",
     "xtend": "~4.0.0"
   },
   "peerDependencies": {
     "proxyquire": "1",
```

Comment:
Given that the devDependency for proxyquire itself was bumped to v2, should this peerDependency also have been expanded to include v2?

(Also thanks for your work on this tool and I'm happy to see it get some maintenance!)

Reply:
Right you are! Will fix that.

Reply:
Fixed in https://github.com/bendrucker/proxyquire-universal/commit/30f0f1920a77278b1b8762e2d5bbe656ac90eac7